libraries/helpers.rb

```diff
@@ -113,8 +113,6 @@ def product_lookup(product, version = '0.0.0')
       elsif (product == 'push-server') && (v < Mixlib::Versioning.parse('2.0.0'))
         data['package-name'] = 'opscode-push-jobs-server'
         data['ctl-command'] = 'opscode-push-jobs-server-ctl'
-      elsif (product == 'push-client') && (v < Mixlib::Versioning.parse('2.0.0'))
```

Comment:
Unless the older opscode prefixed package is elsewhere, I don't see any packages in the packagecloud repos which match the package name `opscode-push-jobs-client`. I have no doubt it exists somewhere, but there likely needs to be more logic if there is another repo.